packages/client/wallets/aa/src/utils/signer.ts

```diff
@@ -27,70 +27,39 @@ type CreateOwnerSignerInput = {
 };
 
 export async function createOwnerSigner({
-    userIdentifier,
-    projectId,
     chain,
     walletConfig,
-    crossmintService,
```

Comment:
if we only use `chain` and `walletConfig` now, then let's update the definition for `CreateOwnerSignerInput` to only those params